### PR TITLE
Added the scroll to top button

### PR DESCRIPTION
**Pull request by abhinav-m22**
## Related Issue

- Information about the related issue

* Closes:- #168 

### Describe the changes you've made

Added the scroll to top nav button at the bottom right corner of the web page.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## How Has This Been Tested?

I have setup the website on my local environment and then run it and tested the changes.

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

|      Original       |      Updated       |
| :-----------------: | :----------------: |
| 
image
 | image |

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/UniKonf/vibey/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by Deepu178 — CHANGES_REQUESTED** [review]
You are using npm for package management but we are using yarn package manger.
You are writing pure css while we are using tailwindCSS.
I would suggest you to make those changes and commit it.

**Comment by arunavabasucom** [comment]
I would like to work on this!

**Comment by abhinav-m22** [comment]
Hey @arunavabasu-03 
I've already made the changes.. There are just some conflicts so I'm resolving it.... Please find some other issue

**Comment by abhinav-m22** [comment]
Hey @Deepu178 
I've made the required changes.. Please check

**Comment by abhinav-m22** [comment]
Hey @Deepu178 
I have made the required changes... Please check

**Comment by parthpathakpp1** [comment]
> Hey @Deepu178 I have made the required changes... Please check

@abhinav-m22 I need some help . Can you share your Discord ID.

**Comment by Deepu178** [comment]
@abhinav-m22 You are still using npm packages to install the dependecies. I would suggest few steps to resolve this error.
1. Delete package-lock.json file
2. Run yarn command
3. Commit the changes
Just follow these steps and it will be ready to merge.

**Comment by abhinav-m22** [comment]
Hey @Deepu178  Thanks for helping me out.
I've followed the steps you told.
But there were many conflicts from my side in the repo... I've resolved them.. But if there are any issues, please let me know.
I'll create a new PR if any problem exists.
Thank you!

**Comment by abhinav-m22** [comment]
@Deepu178 Any updates?

File: src/styles/globals.css [hunk 1]
```diff
@@ -123,3 +123,31 @@ body {
 .ais-SearchBox input {
   @apply h-full w-full rounded-xl border-2 border-primary/30 bg-transparent p-4 text-3xl outline-none placeholder:text-base-content focus:ring-2 focus:ring-primary;
 }
+
```
Comment (Deepu178) [inline src/styles/globals.css:126]: @abhinav-m22  we are using Yarn package manager for installing dependencies and not npm package manager for installation.
Please remove package-lock.json file and install the dependencies using yarn.
Commit those changes.

**Review by Deepu178 — CHANGES_REQUESTED** [review]
@abhinav-m22 you are still using npm package manager for installation and I would suggest you to install dependencies usin yarn.

**Comment by Deepu178** [comment]
@abhinav-m22  Are you still working on this issue?

**Comment by rushabhpanbude** [comment]
> @abhinav-m22 Are you still working on this issue?

You can pass this task to me

**Comment by Deepu178** [comment]
This feature has been added through other pr so looking forward for more contribution from your side.

**Pull request closed without merging**